README.md

```diff
@@ -1,76 +1,72 @@
+# Source
+Inspired by [yu1tiao/PasswordInputView](https://github.com/yu1tiao/PasswordInputView)
 
 ## Features
+1) This library can display plaintext or ciphertext, ciphertext supports displaying dots, asterisks, or any characters.
+2) The user can set the color of the border of the input box and password.
+3) The color of the input box changes after providing input.
+4) The user can set the password style as asterisk, plaintext and circle.
+5) The user can set the border style line and box.
+6) The user can set the maximum length of the password.
+7) You can set the interval and rounded corners between the password boxes.
+8) You can configure the radius for the input boxes.
 
-* 仿支付宝微信风格
-* 下划线风格
-* 可显示明文或者密文，密文支持显示圆点，星号，或者任意字符
-* 设置密码框之间的间隔和圆角(间隔为0时圆角只显示最左和最右的圆角)
-* 设置边框和密码的颜色
-* 支持已输入部分单独设置颜色
+<img src="screenshots/passwordinputview.gif" width="500">
 
-### [关于原理](https://juejin.im/post/5d121b92e51d45108f254230)
-
-## Usages
+## Dependency
+1. For using passwordinputview module in sample app, include the source code and add the below dependencies in entry/build.gradle to generate hap/support.har.
+```groovy
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar', '*.har'])
+    testImplementation 'junit:junit:4.13.1'
+    implementation project(path: ':passwordinputview')
+    ohosTestImplementation 'com.huawei.ohos.testkit:runner:1.0.0.200'
+}
+```
 
-### 只有一个类，直接拷贝到你的项目中，别忘了自定义属性的配置
+2. For using passwordinputview in separate application using har file, add the har file in the entry/libs folder and add the dependencies in entry/build.gradle file.
+```groovy
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.har'])
+    testImplementation 'junit:junit:4.13'
+}
+```
 
-#### 在布局中直接使用
+## Usage
+#### Include following code in your layout:
 ```xml
-    <!--
-         注意EditText的默认背景会有边距，会影响绘制，所以需要去掉
-         android:background="@null"
-     -->
     <com.matthew.passwordinput.lib.PasswordInputView
-        android:layout_width="match_parent"
-        android:layout_height="50dp"
-        android:layout_marginTop="10dp"
-        android:background="@null"
-        android:padding="1dp"
-        android:text="123"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:pwv_haveInputBorderColor="@color/colorAccent"
-        app:pwv_pwdStyle="plaintext"
-        app:pwv_radius="10dp"
-        app:pwv_spacing="12dp" />
-
-```
-#### 设置监听器
-```java
-    passwordView.setInputListener(new PasswordInputView.InputListener() {
-        @Override
-        public void onInputCompleted(String text) {
-            // 输入完成后回调
-        }
-    });
+         ohos:id="$+id:passwordView"
+         ohos:width="match_parent"
+         ohos:height="50vp"
+         ohos:top_margin="10vp"
+         ohos:background_element="@null"
```

Comment:
This will not work in HMOS. Can remove this attribute as its not specific to this library.

Reply:
Done